docs/_articles/en/vscode-desktop/java-setup.md

```diff
@@ -73,54 +30,79 @@ Zulu Java builds are TCK-tested and free to download and use without restriction
 7. Download the installer.
 8. Once the file is downloaded open it and complete the installation steps.
 
-Inside Visual Studio Code, set `salesforcedx-vscode-apex.java.home` to one of the following values.
+### Oracle Java 17 (Officially Tested)
+
+1. Navigate to the [download page](https://www.oracle.com/java/technologies/downloads/).
+2. Navigate to **Java SE Development Kit 17.0.2 downloads**.
+3. Click the download link that applies to your OS.
+4. If prompted to log in, use your Oracle account.
+5. Once the file is downloaded open it and complete the installation steps.
+
+**Note**: Installing from the executable correctly installs Java in the standard Java Home path (C:\Program Files\) making setting the JDK path straightforward.
+
+## Update JDK Path
+
+Update the salesforcedx-vscode-apex.java.home setting to the full pathname of the Java Runtime that is used to launch your Apex server.
+
+1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
```

Comment:
On my Mac, I see my settings at **Code -> Settings -> Settings**.  Is that different from what everyone else is seeing?

Reply:
Good catch. Thank you!